config/routes.rb

```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
+  devise_for :users, controllers: { omniauth_callbacks: 'users/omniauth_callbacks' }, only: [:omniauth_callbacks]
```

Comment:
It's interesting that `user_developer_omniauth_authorize_path`  (and the OIDC variant) are not defined in routes.